### DISCUSSION TITLE
Browser Timeline Behavior

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#118] [Q&A] [answered]
Is it possible to have two or more timelines for browsing(multiple tabs/windows)? I want to have some websites remain open. 
Apart from that, is the agent able to skip/accept/reject cookie policy consents(or popups that say 'the website wants to you send notifications')? Some of them are like popups and nothing happens without accepting/rejecting them.

**Comment by sei-dupdyke** [comment, marked as answer]
You can have multiple handler actions, like for the browser, you can have two randoms with different URL arrays so that one window is browsing from list A and one window is browsing list B.

I don't currently have any way to deal with cookie popups, they are likely unique to each site and would require a great deal of scripting.

**Reply by Cyb3r-Monk** [reply]
I see that the Command: "random" means pick a URL from the list randomly, not rondom HTTP methods(GET, POST, etc.). Am I correct?

**Reply by sei-dupdyke** [reply]
That is correct. For most of the sites we use in an exercise, 99% of the requests are GET.

**Reply by Cyb3r-Monk** [reply]
I used Command:"browse" and gave a list of URLs, it just kept browsing the first URL in the list. This explains why it happened :) 
Thanks, Dustin!
